02-household-queries/chainlit-household-bot.py

```diff
@@ -157,13 +249,17 @@ async def message_submitted(message: cl.Message):
     # 3. Use LlmPrompts lp.register_answer
 
     # Reminder to use make_async for long running tasks: https://docs.chainlit.io/guides/sync-async#long-running-synchronous-tasks
-
     if settings["streaming"]:
         await call_llm_async(message)
```

Comment:
Note that if the user chooses `streaming` AND `use_vector_db`, then the RAG chain will not be called.

Reply:
Add a conditional block like this:
```
if settings["use_vector_db"]:
  await cl.Message("Change the setting to use non-streaming instead").send()
```

Reply:
☝️ Add the warning message in the UI.